projects/storefrontapp-e2e-cypress/cypress/helpers/applied-promotions.ts

```diff
@@ -38,11 +39,10 @@ export function checkForAppliedPromotions() {
 }
 
 export function addProductToCart() {
-  cy.intercept(
-    `${Cypress.env('API_URL')}${Cypress.env('OCC_PREFIX')}/${Cypress.env(
-      'BASE_SITE'
-    )}/users/*/carts/*?fields=DEFAULT,potentialProductPromotions*`
-  ).as('cart_refresh');
+  interceptGet(
+    'cart_refresh',
+    '/users/*/carts/*?fields=DEFAULT,potentialProductPromotions*'
+  );
```

Comment:
Why update the promotions e2e in this PR?

Reply:
I worked on this part. I noticed the tests break when pointing to a local OCC source (due to some error in calculating API_URL). This is to make the set-up consistent and possibly prevent flakiness.